src/cache.cpp

```diff
@@ -812,7 +812,7 @@ std::uint64_t Cache::cleanup_cache(std::vector<std::shared_ptr<RssFeed>> feeds,
 			"Cache::cleanup_cache: NOT cleaning up cache...");
 
 		std::string query = (
-				"SELECT count(rss) "
+				"SELECT distinct rss "
```

Comment:
```suggestion
				"SELECT DISTINCT rss "
```

We put SQL keywords into uppercase to make the query easier to read.

Reply:
Fixed.